src/rqt_publisher/publisher.py

```diff
@@ -137,7 +138,7 @@ def _add_publisher(self, publisher_info):
 
         # create publisher and timer
         publisher_info['publisher'] = self._node.create_publisher(
-            msg_module, publisher_info['topic_name'])
+            msg_module, publisher_info['topic_name'], QoSProfile(depth=10))
```

Comment:
This profile needs to be passed as a keyword argument in order to be compatible with Crystal.

Reply:
done in 
4a48baf